autoPyTorch/pipeline/components/setup/lr_scheduler/CosineAnnealingWarmRestarts.py

```diff
@@ -20,20 +20,19 @@ class CosineAnnealingWarmRestarts(BaseLRComponent):
     restarts in SGDR
 
     Args:
-        T_0 (int): Number of iterations for the first restart
-        T_mult (int):  A factor increases T_{i} after a restart
+        n_restarts (int): Number of restarts. In autopytorch, based
+            on the total budget(epochs) there are 'n_restarts'
+            restarts made periodically.
```

Comment:
I think we already discussed it before, but this parameterization reduces the rank correlation with respect to budget of performance, so Hyperband will not be effective compared to the previous parameterization.